From 42d532e83f03bdb08770c5eebd41fb7dc7fdbbfd Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Sun, 5 Feb 2017 16:45:20 +0200
Subject: [PATCH 48/58] ARM: i.MX6qp: cl-som-imx6: dts: add i.MX6qp dts

1) Added cl-som-imx6 device tree file
2) Added sbc-imx6 device tree file

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 arch/arm/boot/dts/imx6qp-cl-som-imx6.dts | 31 +++++++++++++
 arch/arm/boot/dts/imx6qp-sbc-imx6.dts    | 76 ++++++++++++++++++++++++++++++++
 2 files changed, 107 insertions(+)
 create mode 100644 arch/arm/boot/dts/imx6qp-cl-som-imx6.dts
 create mode 100644 arch/arm/boot/dts/imx6qp-sbc-imx6.dts

diff --git a/arch/arm/boot/dts/imx6qp-cl-som-imx6.dts b/arch/arm/boot/dts/imx6qp-cl-som-imx6.dts
new file mode 100644
index 0000000..7f87244
--- /dev/null
+++ b/arch/arm/boot/dts/imx6qp-cl-som-imx6.dts
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2016 CompuLab Ltd.
+ *
+ * Author: Valentin Raevsky <valentin@compulab.co.il>
+ *
+ * The code contained herein is licensed under the GNU General Public
+ * License. You may obtain a copy of the GNU General Public License
+ * Version 2 or later at the following locations:
+ *
+ * http://www.opensource.org/licenses/gpl-license.html
+ * http://www.gnu.org/copyleft/gpl.html
+ */
+
+/dts-v1/;
+#include "imx6q.dtsi"
+#include "imx6q-cl-som-imx6.dtsi"
+#include "imx6q-cm-fx6.dtsi"
+#include "imx6qp.dtsi"
+
+/ {
+	model = "CompuLab SOM-iMX6";
+	compatible = "compulab,som-imx6", "compulab,cm-fx6", "fsl,imx6qp";
+};
+
+&pcie {
+	status = "okay";
+};
+
+&sata {
+	status = "okay";
+};
diff --git a/arch/arm/boot/dts/imx6qp-sbc-imx6.dts b/arch/arm/boot/dts/imx6qp-sbc-imx6.dts
new file mode 100644
index 0000000..bf15356
--- /dev/null
+++ b/arch/arm/boot/dts/imx6qp-sbc-imx6.dts
@@ -0,0 +1,76 @@
+/*
+* Copyright 2016 CompuLab Ltd.
+*
+* Author: Valentin Raevsky <valentin@compulab.co.il>
+*
+* The code contained herein is licensed under the GNU General Public
+* License. You may obtain a copy of the GNU General Public License
+* Version 2 or later at the following locations:
+*
+* http://www.opensource.org/licenses/gpl-license.html
+* http://www.gnu.org/copyleft/gpl.html
+*/
+
+/dts-v1/;
+#include "imx6q.dtsi"
+#include "imx6q-cl-som-imx6.dtsi"
+#include "imx6q-cm-fx6.dtsi"
+#include "imx6qdl-sb-fx6x.dtsi"
+#include "imx6qdl-sb-fx6.dtsi"
+#include "imx6qp.dtsi"
+
+/ {
+	model = "CompuLab SOM-iMX6 on SB-FX6";
+	compatible = "compulab,som-imx6", "compulab,cm-fx6", "compulab,sbc-imx6", "compulab,sb-fx6", "fsl,imx6qp";
+};
+
+&mxcfb1 {
+	late_init = <1>;
+	prefetch;
+};
+
+&mxcfb2 {
+	prefetch;
+};
+
+&mxcfb3 {
+	prefetch;
+};
+
+&mxcfb4 {
+	prefetch;
+};
+
+&pre1 {
+	status = "okay";
+};
+
+&pre2 {
+	status = "okay";
+};
+
+&pre3 {
+	status = "okay";
+};
+
+&pre4 {
+	status = "okay";
+};
+
+&prg1 {
+	memory-region = <&memory>;
+	status = "okay";
+};
+
+&prg2 {
+	memory-region = <&memory>;
+	status = "okay";
+};
+
+&pcie {
+	status = "okay";
+};
+
+&sata {
+	status = "okay";
+};
-- 
1.9.1

